CHANGELOG.md

```diff
@@ -30,6 +30,11 @@
 - Removed `roster:ingest` command.
 - Removed `roster:assignments:bulk-create` command.
 - Removed `roster:assignments:bulk-cancel` command.
+- User ingester command has been removed.
+
+### Fixed
+- Fixed Code Style issues in `InfrastructureIngesterTest` and `IngesterCommandTest` files.
```

Comment:
This line can be removed, we are not listing code style fixes in changelog

Reply:
sure
p.s.
but that is line from one of your previous PRs :P